workflow-runtime/src/test/java/com/squareup/workflow1/RecordingWorkflowInterceptor.kt

```diff
@@ -8,25 +8,27 @@ import com.squareup.workflow1.WorkflowInterceptor.WorkflowSession
  * Workflow interceptor that records all received events in a list for testing.
  */
 @OptIn(ExperimentalWorkflowApi::class)
-class RecordingWorkflowInterceptor : SimpleLoggingWorkflowInterceptor() {
+internal open class RecordingWorkflowInterceptor : SimpleLoggingWorkflowInterceptor() {
 
   private var events: List<String> = emptyList()
 
   override fun logBeforeMethod(
     name: String,
-    session: WorkflowSession
+    session: WorkflowSession,
+    vararg extras: Pair<String, Any?>
   ) {
     events += "BEGIN|$name"
   }
 
   override fun logAfterMethod(
     name: String,
-    session: WorkflowSession
+    session: WorkflowSession,
+    vararg extras: Pair<String, Any?>
   ) {
     events += "END|$name"
   }
 
-  fun consumeEvents(): List<String> = events
+  private fun consumeEvents(): List<String> = events
```

Comment:
This was a typo? Maybe was used for tests at some point

Reply:
Not a typo, now only used here.